templateflow/conf/__init__.py

```diff
@@ -88,20 +88,24 @@ def _update_datalad():
 
 def init_layout():
     from .bids import Layout
+    from bids.layout import BIDSLayoutIndexer
 
     global TF_LAYOUT
     TF_LAYOUT = Layout(
         TF_HOME,
         validate=False,
         config="templateflow",
-        ignore=[
-            ".git",
-            ".datalad",
-            ".gitannex",
-            ".gitattributes",
-            ".github",
-            "scripts",
-        ],
+        indexer=BIDSLayoutIndexer(
```

Comment:
should not the config go inside the indexer too?

Reply:
I'm not sure here, since BIDSLayout's default is `bids` yet BIDSLayoutIndexer's is `layout_config.json` - cc @effigies

Reply:
No. `config_filename` is a parameter to look for config files that are packaged with the dataset.